test_infra/stacks/databases_stack.py

```diff
@@ -475,13 +477,13 @@ def _setup_mysql_serverless(self) -> None:
                 password=self.db_password_secret,
             ),
             scaling=rds.ServerlessScalingOptions(
-                auto_pause=cdk.Duration.minutes(5),
+                auto_pause=Duration.minutes(5),
                 min_capacity=rds.AuroraCapacityUnit.ACU_1,
                 max_capacity=rds.AuroraCapacityUnit.ACU_1,
             ),
-            backup_retention=cdk.Duration.days(1),
+            backup_retention=Duration.days(1),
             vpc=self.vpc,
-            vpc_subnets=ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE),
+            vpc_subnets=ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE_WITH_NAT),
```

Comment:
Just checking if that is intended difference?

Reply:
Yes, `ec2.SubnetType.PRIVATE` does not exist in cdk v2 and [is marked as deprecated in CDK v1.](https://docs.aws.amazon.com/cdk/api/v1/python/aws_cdk.aws_ec2/SubnetType.html#aws_cdk.aws_ec2.SubnetType.PRIVATE)